src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx

```diff
@@ -200,13 +206,26 @@ export const useLazyConsensusPage = (
             complementaryLabel,
             maxValue,
             validation,
+            accordionItemDescription,
           }) => {
+            const subTitleText = description.defaultMessage
+              ? description.defaultMessage.split('\n')[0]
+              : description;
+
+            let descriptionText: any;
```

Comment:
please add correct type

Reply:
maybe true to use useState and on useEffect change the descriptionText?

Reply:
Removed